README.md

```diff
@@ -10,6 +10,15 @@
   </a>
 </p>
 
+
+> ## ⚠️ Note!
```

Comment:
Lets move this in `docs/README.md` so that it is visible when looking at the gitbook.